plane/source/docs/thermal-mode.rst

```diff
@@ -0,0 +1,16 @@
+.. _thermal-mode:
+
+============
+THERMAL Mode
+============
+
+If SOARING is enabled (:ref:`SOAR_ENABLE<SOAR_ENABLE>` = 1 and it has not been disabled by an RC switch (``RCx_OPTION =88``), this mode will idle the throttle, and begin circling searching for vertical lift.
+Navigation is controlled in the same manner as the previous mode, ie if a pilot controlled mode, it remains so. If an auto-navigating mode, it continues the course demanded by that mode. 
+
+This is the mode that is used for autonomous soaring (See :ref:`soaring-4_1`) and is automatically entered if a thermal is detected during the gliding portion of SOARING flight, and will begin circling searching for the lift center.
+
+Like the THERMAL mode phase of autonomous SOARING it will exit under the same conditions and return to the previous flight mode.
+
+This can be used in modes like FBWA if suddenly entering a thermal to continue in that thermal until THERMAL mode exit conditions are met.
```

Comment:
Confusing, suggest: "This mode can be triggered by the pilot from any mode (like FBWA) using the GCS or RC switch, in which case THERMAL mode will begin searching for lift until THERMAL mode exit conditions are met or the pilot commands a mode change"